### DISCUSSION TITLE
yacd面板怎么能显示机场流量剩余呢

### DISCUSSION THREAD

**Discussion by seankuo1** [kkkgo/PaoPaoGateWay#101] [问和答] [answered]
我是yaml文件中添加了机场订阅链接，在clash verge中可以显示
![1717046746236](https://github.com/kkkgo/PaoPaoGateWay/assets/74480021/555b9c48-0eb0-4c67-9db2-6fad8fdd39c1)

**Comment by kkkgo** [comment, marked as answer]
最新版本已支持显示。https://github.com/kkkgo/PaoPaoGateWay/releases

**Reply by wwushi** [reply]
直接把订阅链接放到custom.yaml文件里面就行？

**Reply by seankuo1** [reply]
> 直接把订阅链接放到custom.yaml文件里面就行？

改写下你的YAML文件，用proxy-provider
proxy-providers:

  机场1:
    type: http
    url: xxxxxxxxxxxxxxxxxxxxxxx
    interval: 172800
    path: ./机场1.yaml
    health-check:
      enable: true
      url: http://www.gstatic.com/generate_204
      interval: 300

proxy-groups:
  - name: 🧱International
    type: select
    proxies:
    - ♻️AutoTest
    - ⛩️Home mode
    - 🔫Free mode
    - 🚀Direct
    use:
    - 机场1
  - name: 𝕏(Twitter)
    type: select
    proxies:
    - 🧱International
    - ⛩️Home mode
    - 🔫Free mode
    - 🚀Direct
    use:
    - 机场1
